app/forms/submissions/api_trn_validator.rb

```diff
@@ -4,16 +4,6 @@ module Submissions
   class ApiTrnValidator < BaseValidator
     include FundingHelper
 
-    class_attribute :extra_validators, instance_writer: false, default: {}
-
-    class << self
-      def missing_data_validator(name, options)
-        extra_validators[name] = options
-      end
-    end
-
-    missing_data_validator :placements, form: "PlacementDetailForm", if: :requires_placements?
```

Comment:
We don't need to validate the placements?

Reply:
app/forms/placement_form.rb is used for validation already as far as I can tell 🤔

Reply:
Yeah, although not quite obvious is been called through the `Submissions::TrnValidator` at [trainee.rb:598](https://github.com/DFE-Digital/register-trainee-teachers/blob/6977-apitrnvalidator-not-being-used/app/models/trainee.rb#L598)

Reply:
Yeah, although not quite obvious is been called through the Submissions::TrnValidator at [trainee.rb:598](https://github.com/DFE-Digital/register-trainee-teachers/blob/6977-apitrnvalidator-not-being-used/app/models/trainee.rb#L598)